code/game/objects/items/rcd/RPD.dm

```diff
@@ -259,6 +263,13 @@ GLOBAL_LIST_INIT(transit_tube_recipes, list(
 	UnregisterSignal(user, COMSIG_MOUSE_SCROLL_ON)
 	return ..()
 
+/obj/item/pipe_dispenser/proc/pipe_layer_bits2num(layerbits)
```

Comment:
1. This proc name is a mouthful. What does it do?
2. Don't abbreviate names, and use snake_case.